sgxs-tools/src/sgx_detect/main.rs

```diff
@@ -167,7 +168,7 @@ struct FailTrace<'a>(pub &'a Error);
 impl<'a> fmt::Display for FailTrace<'a> {
     fn fmt(&self, fmt: &mut fmt::Formatter) -> fmt::Result {
         write!(fmt, "{}", self.0)?;
-        for cause in self.0.causes() {
+        for cause in self.0.iter_chain() {
```

Comment:
I don't think this is correct. This includes `self.0` which was already printed.